__tests__/login/popUp/PopUpLoginHandler.spec.ts

```diff
@@ -55,49 +54,33 @@ describe("PopUpLoginHandler", () => {
       const handler = getInitialisedHandler();
       expect(
         await handler.canHandle({
+          sessionId: "mySession",
           popUp: true,
-          popUpRedirectPath: "/redirect"
+          redirectUrl: new URL("/redirect")
         })
       ).toBe(true);
     });
     it("Rejects a configuration not for popup", async () => {
       const handler = getInitialisedHandler();
       expect(
         await handler.canHandle({
+          sessionId: "mySession",
           popUp: false,
-          redirect: new URL("https://coolsite.com/redirect")
+          redirectUrl: new URL("https://coolsite.com/redirect")
         })
       ).toBe(false);
     });
   });
 
   describe("handle", () => {
-    it("can handle popup", async () => {
-      const { open } = window;
-      delete window.open;
-      const popUpWindowObject = {
-        closed: false
-      } as Window;
-      window.open = jest.fn(() => popUpWindowObject);
-
+    it("is unimplemented", async () => {
       const handler = getInitialisedHandler();
-      setTimeout(() => {
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        (popUpWindowObject as any).closed = true;
-      }, 1000);
-      const session = await handler.handle({
-        popUp: true,
-        popUpRedirectPath: "/redirect"
-      });
-      expect(window.open).toHaveBeenCalledWith(
-        (LoginHandlerResponse.neededAction as INeededRedirectAction)
-          .redirectUrl,
-        "Log In",
-        "resizable,scrollbars,width=500,height=500,"
-      );
-      expect(session).toBe(SessionCreatorCreateResponse);
-
-      window.open = open;
+      await expect(
+        handler.handle({
+          sessionId: "mySession",
```

Comment:
More of a thought experiment really (i.e. I'd certainly leave it as a simple string for now), but in principle, given we're Turtles-all-the-way-down, shouldn't **_all_** our ID's be IRIs really? It's kinda interesting to think that my SessionID could actually be de-referenceable, or that it could be persisted in a triplestore along with meta-data associated with the Session itself (i.e. I had this session open from time X to time Y, and I was using this device, and I 'fetched' these Pod resources, etc.)... seems interesting to me...?